src/go/k8s/kuttl-test.yaml

```diff
@@ -11,11 +11,19 @@ kindConfig: ./kind.yaml
 kindNodeCache: false
 commands:
   - command: "kubectl taint nodes -l node-role.kubernetes.io/master= node-role.kubernetes.io/master:NoSchedule-"
+    ignoreFailure: true
   - command: "./hack/install-cert-manager.sh"
+    background: true
+    ignoreFailure: true
   - command: "kubectl create -f https://raw.githubusercontent.com/prometheus-operator/prometheus-operator/e23ff77fceba6a5d9f190f5d1a123c87701dc964/bundle.yaml"
-  - command: "sh -c 'kustomize build config/e2e-tests | kubectl apply -f -'"
+    background: true
+    ignoreFailure: true
+  - command: "sh -c 'until kustomize build config/e2e-tests | kubectl apply -f -; do sleep 0.5; done'"
+    background: true
+    ignoreFailure: true
```

Comment:
curious why ignoreFailure?

Reply:
Yes, there's a slim chance this would fail when run on a new cluster, but it's extremely unlikely. What this does do is allow us to run kuttl with the --skip-cluster-delete flags when testing things locally, saving the several minutes it takes to bootstrap a new cluster.